README.md

```diff
@@ -1,16 +1,16 @@
-<h1 align="center">Welcome to cookies clean winforms GUI template 👋</h1>
+<h1 align="center">Welcome to MJs Clean KeyAuth GUI 👋</h1>
 
 <p align="center">
   <img src="https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000" >
   <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" >
   <img src="https://img.shields.io/badge/license-MIT-yellow.svg" >
-  <img src="https://img.shields.io/github/last-commit/Callumgm/Clean-GUI-Template">
+  <img src="https://img.shields.io/github/last-commit/MJMODZZ/Clean-KeyAuth-GUI">
   <a href="https://twitter.com/Flashouttt" target="_blank">
     <img src="https://img.shields.io/twitter/follow/Flashouttt.svg?style=social">
   </a>
 </p>
 
-> Simple and clean WinForms GUI created in my own spare time
+> Cool looking KeyAuth Key System GUI
```

Comment:
Change this back to

Simple and clean WinForms GUI created in my own spare time